internal/handlers/common/aggregations/operators/operators_errors.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package operators provides aggregation operators.
+package operators
+
+// operatorErrorCode represents the type of error.
+type operatorErrorCode uint
+
+const (
+	_ operatorErrorCode = iota
+
+	// ErrArgsInvalidLen indicates that operator have invalid amount of arguments.
+	ErrArgsInvalidLen // Expression takes different amount of arguments.
+
+	// ErrWrongType indicates that operator field is not a document.
+	ErrWrongType // Invalid type of operator field (expected document)
+
+	// ErrEmptyField indicates that operator field does not specify any operator.
```

Comment:
```suggestion
	// ErrEmptyField indicates that operator field is empty.
```

Suggesting this because `operator field does not specify any operator` seems to indicate ErrNoOperator